### PR TITLE
Deepalakshmibits patch 2

### PR DESCRIPTION
**Pull request by Deepalakshmibits**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 ## Group members
 1. Kartik Rajesh - 2021mt93208
+2. Deepalakshmi S - 2021mt93148
```
Comment (Deepalakshmibits) [inline README.md]:
```suggestion
2. Deepalakshmi S - 2021mt93148
```

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
...
```
Comment (Deepalakshmibits) [inline README.md:5]:
```suggestion
2. Deepalakshmi S - 2021MT93148
```

Comment (Deepalakshmibits) [inline README.md:5]:
```suggestion
2. Deepalakshmi S - 2021MT93148
```

**Comment by kartik70rajesh** [comment]
Redundant PR so closing it.

**Pull request closed without merging**